### PR TITLE
Add example and notice about task type Stored Procedure

### LINKED ISSUE #8019: [Feature][Document] Add example and notice about task type Stored Procedure
Due to our tasks task description is not containing some detail, and I think we should add a new section notice and give an example to the task stored-procedure in the document https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/guide/task/stored-procedure.html

### PR DESCRIPTION
**Pull request by hstdream**
## Purpose of the pull request

this pr close: #8019 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by hstdream** [comment]
this pr close: #8019

File: docs/docs/zh/guide/task/stored-procedure.md [hunk 1]
```diff
@@ -7,6 +7,33 @@
...
  </p>
 
-- 数据源：存储过程的数据源类型支持MySQL和POSTGRESQL两种，选择对应的数据源
-- 方法：是存储过程的方法名称
-- 自定义参数：存储过程的自定义参数类型支持IN、OUT两种，数据类型支持VARCHAR、INTEGER、LONG、FLOAT、DOUBLE、DATE、TIME、TIMESTAMP、BOOLEAN九种数据类型
+- 节点名称：设置任务节点的名称，一个工作流定义中的节点名称是唯一的。
+- 运行标志：标识这个结点是否能正常调度，如果不需要执行，可以打开禁止执行开关。
+- 描述：描述该节点的功能。
+- 任务优先级：worker 线程数不足时，根据优先级从高到低依次执行，优先级一样时根据先进先出原则执行。
+- Worker 分组：任务分配给 worker 组的机器执行，选择Default，会随机选择一台 worker 机执行。
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/task/stored-procedure.md]: Your Chinese docs should also use the highlight as English `Default`, just like your doc in English.

**Review by zhongjiajie — COMMENTED** [review]
BTW, please use the English screenshot instead of Chinese one

File: docs/configs/docsdev.js [hunk 2]
```diff
@@ -433,7 +433,7 @@ export default {
                                 link: '/zh-cn/docs/dev/user_doc/guide/task/dependent.html',
                             },
                             {
-                                title: 'Stored Procedure',
+                                title: 'PROCEDURE',
```
Comment (zhongjiajie) [inline docs/configs/docsdev.js:753]:
```suggestion
                                title: 'Procedure,
```

File: docs/configs/docsdev.js [hunk 3]
```diff
@@ -119,7 +119,7 @@ export default {
                                 link: '/en-us/docs/dev/user_doc/guide/task/dependent.html',
                             },
                             {
-                                title: 'Stored Procedure',
+                                title: 'PROCEDURE',
```
Comment (zhongjiajie) [inline docs/configs/docsdev.js:109]:
```suggestion
                                title: 'Procedure',
```

File: docs/configs/docsdev.js (hunk 2 above)
Comment (zhongjiajie) [reply docs/configs/docsdev.js:753]: If you change the sidebar title, you should also change the title in content `Stored Prcedure`

File: docs/docs/en/guide/task/stored-procedure.md [hunk 4]
```diff
@@ -8,6 +8,32 @@
    <img src="/img/procedure-en.png" width="80%" />
  </p>
 
-- DataSource: The DataSource type of the stored procedure supports MySQL and POSTGRESQL, select the corresponding DataSource.
-- Method: The method name of the stored procedure.
-- Custom parameters: The custom parameter types of the stored procedure support `IN` and `OUT`, and the data types support: VARCHAR, INTEGER, LONG, FLOAT, DOUBLE, DATE, TIME, TIMESTAMP and BOOLEAN.
\ No newline at end of file
+- Node name: The node name in a workflow definition is unique.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/stored-procedure.md]: you should change the document like https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/task/sql.md. We have section `Overview ` and section `Create DataSource`

File: docs/docs/en/guide/task/stored-procedure.md [hunk 5]
```diff
@@ -8,6 +8,32 @@
...
+- Descriptive information: Describe the function of the node.
+- Task priority: When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order.
+- Worker grouping: Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution.
+- Environment Name: Configure the environment name in which run the script.
+- Times of failed retry attempts: The number of times the task failed to resubmit.
+- Failed retry interval: The time interval (unit minute) for resubmitting the task after a failed task.
+- Delayed execution time: The time (unit minute) that a task delays in execution.
+- Timeout alarm: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
+- DataSource: The DataSource type of the stored procedure supports MySQL,POSTGRESQL,HIVE,SPARK,CLICKHOUSE,ORACLE,SQLSERVER,DB2 and PRESTO, select the corresponding DataSource.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/stored-procedure.md]:
```suggestion
- DataSource: The DataSource type of the stored procedure supports MySQL,POSTGRESQL,HIVE,SPARK,CLICKHOUSE,ORACLE,SQLSERVER,DB2 and PRESTO.
```

File: docs/docs/zh/guide/task/stored-procedure.md (hunk 1 above)
Comment (zhongjiajie) [reply docs/docs/zh/guide/task/stored-procedure.md]: `Default`

File: docs/docs/en/guide/task/stored-procedure.md (hunk 4 above)
Comment (zhongjiajie) [reply docs/docs/en/guide/task/stored-procedure.md]: https://raw.githubusercontent.com/apache/dolphinscheduler/dev/docs/docs/en/guide/task/sql.md @hstdream

**Comment by caishunfeng** [comment]
@hstdream Please resolve conflicts.

File: docs/docs/en/guide/task/stored-procedure.md [hunk 6]
```diff
@@ -1,4 +1,10 @@
-# Stored Procedure
+# PROCEDURE
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/stored-procedure.md]:
```suggestion
# Procedure
```

File: docs/docs/zh/guide/task/stored-procedure.md [hunk 7]
```diff
@@ -1,12 +1,47 @@
 # 存储过程节点
 
+## 综述
+
+`PROCEDURE` 任务类型，用于执行存储过程的程序
+
+## 创建任务
+
 - 根据选择的数据源，执行存储过程。
 > 拖动工具栏中的![PNG](https://analysys.github.io/easyscheduler_docs_cn/images/toolbar_PROCEDURE.png)任务节点到画板中，如下图所示：
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/task/stored-procedure.md]: Please change this png link from `analysys.github.io` to our local file like you add then in its `Task Example` section

File: docs/configs/docsdev.js [hunk 8]
```diff
@@ -470,7 +470,7 @@ export default {
                                 link: '/zh-cn/docs/dev/user_doc/guide/task/dependent.html',
                             },
                             {
-                                title: 'Stored Procedure',
+                                title: 'Procedure,
                                 link: '/zh-cn/docs/dev/user_doc/guide/task/stored-procedure.html',
```
Comment (zhongjiajie) [inline docs/configs/docsdev.js]: I suggest we should change the task file name from `stored-procedure` to `procedure` to keep the file name same as the content title of first line
```suggestion
                                link: '/zh-cn/docs/dev/user_doc/guide/task/procedure.html',
```

File: docs/docs/en/guide/task/stored-procedure.md [hunk 9]
```diff
@@ -1,4 +1,10 @@
...
+
+## Overview
+
+`PROCEDURE` task type, the program used to execute the stored procedure
+
+## Create Task
 
 - Execute the stored procedure according to the selected DataSource.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/stored-procedure.md]: We should also add how to create task like others `- Drag from the toolbar  to the canvas.`

File: docs/configs/docsdev.js (hunk 8 above)
Comment (zhongjiajie) [reply docs/configs/docsdev.js]: Please also change the EN document

**Review by zhongjiajie — COMMENTED** [review]
* I think we should also add `## Notice` section to notice users they could only call Stored Procedure in this task, if they want to call Stored function, they should use SQL task
* Please change your screenshot from Chinese to English

**Comment by zhongjiajie** [comment]
@hstdream CI failed, it is seem your have some unused imgs, and some deadlink in your change

**Comment by zhongjiajie** [comment]
Hi @hstdream CI still failed, you should remove the images '/img/tasks/demo/procedure_task1.png', '/img/procedure-en.png', '/img/procedure_edit.png', '/img/tasks/demo/procedure_task2.png'  in `img` directory.

And change the screenshot `docs/img/tasks/demo/procedure_task01.png` `docs/img/tasks/demo/procedure_task02.png` language from Chinese to English

**Review by zhongjiajie — COMMENTED** [review]
Thanks for your contribution, but please switch dolphinscheduler language to English for the screenshot, we only accept English version

**Comment by caishunfeng** [comment]
I close this pr due to already implemented in dev branch.

**Pull request closed without merging**